desktop_version/src/FileSystemUtils.cpp

```diff
@@ -585,7 +590,7 @@ static void PLATFORM_getOSDirectory(char* output)
 	SHGetFolderPathW(NULL, CSIDL_PERSONAL, NULL, SHGFP_TYPE_CURRENT, utf16_path);
 	WideCharToMultiByte(CP_UTF8, 0, utf16_path, -1, output, MAX_PATH, NULL, NULL);
 	SDL_strlcat(output, "\\VVVVVV\\", MAX_PATH);
-	CreateDirectory(output, NULL);
+	mkdir(output, 0);
```

Comment:
Nit:
```suggestion
	mkdir(output, 0777);
```

Otherwise lgtm.